tasks/copy_pyqt_to_venv.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eu
+
+if [[ ! -z "${VIRTUAL_ENV:-}" ]]; then
+    echo -e "copying system's pyqt4 and sip into current python venv ${VIRTUAL_ENV}..."
+    python_version=`python --version 2>&1`
+    package_dir=`find "${VIRTUAL_ENV}/lib" -name "site-packages"`
+    if [[ ${python_version} == *"2.7"* ]]; then
+        cp /usr/lib/python2.7/dist-packages/{sipconfig*.py,sip*.so} "${package_dir}"
```

Comment:
If we're in a virtualenv, aren't these libraries copied to the virtualenv?  On MacOS at least they would get installed to `./py2/lib/python2.7/site-packages/PySide/`?  Incidentally, this required launching the console like this `DYLD_LIBRARY_PATH=./py2/lib/python2.7/site-packages/PySide/  python .../consol.py ...`.

Reply:
the point is to copy them to venv so the tox gui-test run can be self-contained without using site-packages option. This hack is currently not used for mac so doesn't matter there.

Reply:
I'm wondering if they're already there, but maybe in a different directory?  If not, then no worries, otherwise we might be able to solve this issue by setting LD_LIBRARY_PATH when invoking tox (though it does seem a bit confusing that `LD_LIBRARY_PATH` would even be needed).

Reply:
I don't think they are available in the venv without this (or system-site-packages which would break the venv isolation a bit).

Reply:
Maybe a bit more portable/robust solution like `python -c "import site; print(site.getsitepackages())"`?

Reply:
at least this particular method is not available under python 2.7. Maybe there could be something else useful in this `site` module but on the other hand this script is in bash...  Need to think about it.